Source/Core/Resource.js

```diff
@@ -1902,7 +1902,7 @@ import TrustedServers from './TrustedServers.js';
 
         // Specifically use the Node version of require to avoid conflicts with the global
         // require defined in the built version of Cesium.
-        var nodeRequire = global.require; // eslint-disable-line
+        var nodeRequire = global.require ? global.require : require; // eslint-disable-line
```

Comment:
Now that we use ES6, you can just remove the `nodeRequire` variable and comment above it and just replace it with `require` directly in this function.

This will most likely have to all change again for Node 13, but there are bigger issues that need to be addressed there too.